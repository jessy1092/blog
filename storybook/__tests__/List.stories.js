
import React from 'react';
import { mount } from 'enzyme';
import expect from 'expect';

import {
	storiesOf,
	describe,
	it,
	specs,
} from '../facade';

import List from '../../src/components/List';

const items = [{
	key: 1, value: 'Lee',
}, {
	key: 2, value: 'Mike',
}];

const stories = storiesOf('List', module);

stories.add('with items', () => {
	const list = (
		<List items={items} />
	);

	specs(() => describe('List', () => {
		it('should have items property', () => {
			const component = mount(list);
			expect(component.props().items).toEqual(items);
		});
	}));

	return list;
});
