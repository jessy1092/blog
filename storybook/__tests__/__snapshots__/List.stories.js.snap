exports[`test with items 1`] = `
<ul>
  <li>
    Lee
  </li>
  <li>
    Mike
  </li>
</ul>
`;
